game_for_class.py

```diff
@@ -4,10 +4,17 @@
 from queue import PriorityQueue
 from dataclasses import dataclass, field
 
+
 # Start with the player's class
 class Character:
     """All the methods the game player can use"""
-    def __init__(self, name, hp, attack, defense, speed, team):
+    def __init__(self,
```

Comment:
Question, Why do I need to put the int and float identifiers

Reply:
Because it enforced type constraints and documents what the method is expecting as inputs.  

It's easier to reason about the code if you understand what all the values are, and the type of the value is one of the most important things to know.